haystack/document_store/sql.py

```diff
@@ -326,6 +326,12 @@ def write_labels(self, labels, index=None):
 
         labels = [Label.from_dict(l) if isinstance(l, dict) else l for l in labels]
         index = index or self.label_index
+        ids_exist_in_db: list = []
+        labels_found = self.get_labels_by_id(ids=[label.id for label in labels], index=index)
```

Comment:
Is there a specific reason why we need this new method? Could we use DocumentStore.get_all_labels() instead?

Reply:
@brandenchan  ` DocumentStore.get_all_labels()` is returning all labels records from Database. Here we need records whose IDs already exist in DB. If labels records are huge then it will slow down execution. 

Is you still think its ok to DocumentStore.get_all_labels(). I will use it.

Reply:
I think that for most cases `DocumentStore.get_all_labels()` is fine and so I would go for this. My concern is that in cases where we have a lot of labels, `get_labels_by_id()` might be slow because we are providing a lot of different ids for the the document store to filter by.

But it also just occurred to me that I think we're still missing something here. We check whether the labels that we're writing share an id with the labels in the document store. But we haven't checked yet whether there are any duplicate ids in the list of labels that we are about to write. We still need to check for this:
```
ids = [l.id for l in labels]
duplicates_exist = len(ids) == len(set(ids))
``` 
We might want to use a `collections.Counter` object to see which ids have duplicates so that we can print these out in the `logger.warning()`

Reply:
Ok. I will fix this bug.